docs/APIReference-Renderer.md

```diff
@@ -0,0 +1,187 @@
+      <pre>environment</pre>
+      The API for Relay core that provides methods for fetching/mutating data, an in-memory cache, etc.
+    </a>
+  </li>
+    <li>
+    <a href="#render">
+      <pre>render</pre>
+      Called to render when data requirements are being fulfilled.
+    </a>
+  </li>
+  <li>
+    <a href="#onreadystatechange">
+      <pre>onReadyStateChange</pre>
+    </a>
+  </li>
+</ul>
+
+## Props
+
+### Container
+
+```
+Container: RelayContainer
+```
+
+Must be a valid `RelayContainer`. Relay will attempt to fulfill its data requirements before rendering it.
+
+
+### forceFetch
+
+```
+forceFetch: boolean
+```
+
+If supplied and set to true, a request for data will always be made to the server regardless of whether data on the client is available to immediately fulfill the data requirements.
+
+
+### onForceFetch
+
+```
+onForceFetch(
+  querySet: RelayQuerySet,
+  callback: (readyState: ReadyState) => void
+): Abortable
+```
+
+This callback prop is called whenever a forceFetch occurs.
+
+
+### onPrimeCache
+
+```
+onPrimeCache(
+  querySet: RelayQuerySet,
+  callback: (readyState: ReadyState) => void
+): Abortable
+```
+
+This callback prop is called whenever a primeCache call occurs.
+
+### QueryConfig
+
+```
+queryConfig: RelayRoute
+```
+
+Either an instance of `Relay.Route` or an object with the `name`, `queries`, and optionally the `params` properties.
+
+
+### Environment
+
+```
+environment: RelayEnvironment
+```
+
+Either `Relay.Store`, an instance of `Relay.Evironment` or an object with the `forceFetch`, `getFragmentResolver`, `getStoreData`, `primeCache`, and `read` function defined.
+
+
+### render
+
+```
+render(
+  props: ?{[propName: string]: mixed},
+  done: boolean,
+  error: ?Error,
+  retry: ?Function,
+  stale: boolean
+): ?RelayRenderCallback
+```
+
+Whenever anything is rendered `render` is called to render the view. If the callback is not supplied, the default behavior is to render the container if data is available, the existing view if one exists, or nothing.
+
+If the callback returns `undefined`, the previously rendered view (or nothing if there is no previous view) is rendered (e.g. when transitioning from one `queryConfig` to another).
+
+#### Example
+
+```{4-6}
+// This is the render function that `RelayRootContainer` defines with `renderFailure, renderFetched, Component, renderLoading` being passed in as props.
+<RelayRenderer
+  Container={ProfilePicture}
+  route={profileRoute}
+  render={({done, error, props, retry, stale}) => {
+        if (error) {
+          if (renderFailure) {
+            return renderFailure(error, retry);
+          }
+        } else if (props) {
+          if (renderFetched) {
+            return renderFetched(props, {done, stale});
+          } else {
+            return <Component {...props} />;
+          }
+        } else {
+          if (renderLoading) {
+            return renderLoading();
+          }
+        }
+        return undefined;
+      }}
+/>
+```
```

Comment:
Not sure how much value this example adds. Thinking forward to an imminent time when `RelayRootContainer` is deprecated and most people using Relay won'd have ever heard of it, the references to and comparisons with `RelayRootContainer` won't be super useful (in fact, they might be distracting and therefore detrimental).